### DISCUSSION TITLE
I get errors when trying to run the plugin, any ideas?

### DISCUSSION THREAD

**Discussion by ChristianRic** [cmp-nct/Stewardess#2] [General]
Hello! I get the following errors when running the plugin:

Error compiling script.
C:\Users\CC12\AppData\Roaming\JetBrains\IdeaIC2021.2\plugins\live-plugins\Stewardess\plugin.kts:121:31: error: unresolved reference: allSuperClasses
    val allFields = javaClass.allSuperClasses().flatMap { it.declaredFields.toList() }
                              ^
C:\Users\CC12\AppData\Roaming\JetBrains\IdeaIC2021.2\plugins\live-plugins\Stewardess\plugin.kts:121:62: error: unresolved reference: declaredFields
    val allFields = javaClass.allSuperClasses().flatMap { it.declaredFields.toList() }
                                                             ^
C:\Users\CC12\AppData\Roaming\JetBrains\IdeaIC2021.2\plugins\live-plugins\Stewardess\plugin.kts:124:43: error: unresolved reference: name
    val field = allFields.find { list+=it.name+","; (it.name == name && (fieldClass.isAssignableFrom(it.type))&&false) }
                                          ^
C:\Users\CC12\AppData\Roaming\JetBrains\IdeaIC2021.2\plugins\live-plugins\Stewardess\plugin.kts:124:57: error: unresolved reference: name
    val field = allFields.find { list+=it.name+","; (it.name == name && (fieldClass.isAssignableFrom(it.type))&&false) }
                                                        ^
C:\Users\CC12\AppData\Roaming\JetBrains\IdeaIC2021.2\plugins\live-plugins\Stewardess\plugin.kts:124:105: error: unresolved reference: type
    val field = allFields.find { list+=it.name+","; (it.name == name && (fieldClass.isAssignableFrom(it.type))&&false) }

**Comment by cmp-nct** [comment]
Is it possible you set the wrong scripting language in Live Plugin ?
That Plugin supports Groovy too.
![image](https://user-images.githubusercontent.com/78893154/215268245-b80f35e1-647f-4445-bc70-285a8a4bf773.png)

I recommend to remove the Stewardess plugin, then to click on + and as in screenshot select Kotlin Plugin.
Then just copy/paste the complete code from the repository here to the new script.
Then click on Run and it should work

I am using in pyCharm and PhpStorm
